devices/ePaper/README.md

```diff
@@ -0,0 +1,111 @@
+﻿# nanoFramework.Iot.Device.ePaper
```

Comment:
Markdown 101: you need an extra line after a title. Also a suggestion for a better title

```suggestion
﻿# ePaper drivers for .NET nanoFramework
﻿
```